cmd/svcat/instance/get_cmd.go

```diff
@@ -85,7 +118,61 @@ func (c *getCmd) get() error {
 		return err
 	}
 
+	if c.planFilter != "" {
+		if !c.filterByPlan(instance) {
+			// Found instances was filtered out by plan
+			return nil
+		}
+	}
+
+	if c.classFilter != "" {
+		if !c.filterByClass(instance) {
+			// Found instances was filtered out by class
+			return nil
+		}
+	}
+
 	output.WriteInstance(c.Output, c.outputFormat, *instance)
 
 	return nil
 }
+
+func (c *getCmd) filterListByPlan(instanceList *v1beta1.ServiceInstanceList) *v1beta1.ServiceInstanceList {
+	p := v1beta1.ServiceInstanceList{
+		Items: []v1beta1.ServiceInstance{},
+	}
+
+	for _, instance := range instanceList.Items {
+		if !c.filterByPlan(&instance) {
+			continue
+		}
+
+		p.Items = append(p.Items, instance)
+	}
+
+	return &p
+}
+
+func (c *getCmd) filterByPlan(instance *v1beta1.ServiceInstance) bool {
```

Comment:
The name of this method isn't ideal. It's not clear whether it returns `true` or `false` when the `instance`'s plan matches the filter. Can you rename this to something like `acceptedByPlanFilter` or `planFilterAccepts`?